SimpleCalculator

```diff
@@ -0,0 +1,134 @@
+//HTML CODE FOR THE CALCULATRO
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
```

Comment:
There is no use of this `<link rel='stylesheet' href="style.css"`(This used for external css)
Because you already wrote the CSS content in the same HTML file. (Internal CSS)